src/main/java/core/basesyntax/AbstractFigure.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+public abstract class AbstractFigure implements Figure {
```

Comment:
Don't begin class or method implementation with an empty line. Remove the redundant empty line at the start of the class definition.